w3-aggregation.md

```diff
@@ -343,10 +345,13 @@ type AggregateOfferDetail struct {
 
 type Offer [PieceInfo]
 
+# Adapted from `PieceInfo` type in filecoin
 # https://github.com/filecoin-project/go-state-types/blob/1e6cf0d47cdda75383ef036fc2725d1cf51dbde8/abi/piece.go#L47-L50
+# Uses `height` field instead of `size`. `height` field can be used to derive `leafCount` and consequently `size`, while
+# allowing the usage of smaller numbers instead of `bigint`.
 type PieceInfo {
```

Comment:
@Gozala by this change, I am considering to just rename `PieceInfo` to `PieceProof` instead.

Motivation would be to create a new name instead of relying on `PieceInfo` that is already known to have `cid` + `size`. We could just name it `PieceProof` and add comment saying it can be used to derive `PieceInfo`. What do you think?

Reply:
IMO it's fine to leave as `PieceInfo`, the point is for this information to now be the "piece information" as we'd not materialize and pass around a size value unless for display.

Reply:
I really don't want to call it a `PieceProof` as it can be confused with `InclusionProof` (in fact that's how it a appears in the go implementation).

If you want to rename perhaps call it a `PieceDetail`, personally I don't have strong feelings about the name. In fact it is a mistake that height is not part of the CID, but I'm not sure it's worth trying to fix that.